src/main/java/racingcar/domain/RacingCarGame.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.domain;
+
+import racingcar.utils.RacingCarUtils;
+import racingcar.view.InputView;
+
+import java.util.List;
+
+public class RacingCarGame {
+    private final Cars cars;
+    private int tryCount;
+    private boolean isRacing = true;
+
+    public RacingCarGame() {
+        cars = new Cars(getCarNames());
+        tryCount = new TryCount(InputView.getTryCountInput()).value();
+    }
+
+    public Cars getCars() {
+        return cars;
```

Comment:
View에 Cars객체를 넘기면 Car에서 다른 메소드를 호출해 내부 상태를 변경 할 가능성이 있으니 Cars 의 getCars() 메소드의 결과를 리턴하면 어떨까요? 

```suggestion
        return cars.getCars()t;
```

Reply:
이게 맞는것 같아요. 
이부분도 디미터의 법칙이 위반 될 수 있는 부분인것 같네요